pkg/storage/sqlstorage/migrates/17-optimized-segments/sqlite.sql

```diff
@@ -0,0 +1,27 @@
+--statement
```

Comment:
The code checks postgres usage, so the postings table will not be used with sqlite.
Perhaps we could remove this migration for sqlite or maybe implements sqlite support on the queries.

Reply:
indeed, I removed the sqlite migration and usage of the postings table